### ISSUE TITLE
For create_h5_dataset.py ，Can the size of the validation and test sets be set to 448x448?

### ISSUE THREAD

**Issue by JUSTDODoDo** [abhaydoke09/Bilinear-CNN-TensorFlow#30]
hello,abhaydoke09:
    Thank you very much for this work.I have a question,For create_h5_dataset.py ，Can the size of the validation and test sets be set to 448x448? The code is 224X224, but I think the saved test set is not too small.Can you give me some  advice?thanks a lot

**Comment by JUSTDODoDo** [comment]
In addition, I still have a problem. After running the second part of the whole model, I will finish training. It seems that the final model is not saved in the code. Why is this done in the absence of the training model? Can you give me some details?

**Comment by MBB001** [comment]
hi，Have you solved your problem？

**Comment by abhaydoke09** [comment]
@JUSTDODoDo @MBB001 

You can use test set to be 448x448. If you look at the output of bilinear layer, `self.phi_I = tf.einsum('ijkm,ijkn->imn',self.conv5_3,self.conv5_3)` it does not matter what the size of image is as long as images fed to both networks are of same size. So you can use 448x448.
